packages/gatsby-cli/src/reporter/loggers/yurnalist/index.ts

```diff
@@ -93,15 +93,27 @@ export function initializeYurnalistLogger(): void {
           activities[action.payload.id] = activity
         } else if (action.payload.type === ActivityTypes.Progress) {
           const bar = new ProgressBar(
-            ` [:bar] :current/:total :elapsed s :percent ${action.payload.text}`,
+            ` [:bar] :current/:total :elapsed s :rate /s :percent ${action.payload.text}`,
```

Comment:
Nit (feel free to ignore)
```suggestion
            ` [:bar] :current/:total :elapseds :rate/s :percent ${action.payload.text}`,
```

extra space there make it look a bit weird (like it was something separate) - this change does in fact result in
```diff
-[=====-------------------------] 18/100 1.5 s 12 /s 18% test #1
+[=====-------------------------] 18/100 1.5s 12/s 18% test #1
```
"postfixes" are "attached" to their numbers which to me make it more clear what those are (IMO)

but up to you and feel free to ignore